syz-cluster/dashboard/templates/base.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="/static/bootstrap.min.css">
+    <title>Syz-Cluster</title>
+  </head>
+  <body>
+    <nav class="navbar navbar-light bg-light justify-content-start">
+      <a class="navbar-brand px-3" href="#">Syz-Cluster</a>
```

Comment:
Syz-Cluster looks different. Syzbot, syzkaller, syz-manager etc...
Let's syz-cluster. Syz-cluster because it is the beginning of the line.